vmdb/app/helpers/container_route_helper/textual_summary.rb

```diff
@@ -0,0 +1,45 @@
+module ContainerRouteHelper
```

Comment:
changing this line to module ContainerProjectHelper::TextualSummary and removing next line fixes the error in my previous screenshot same for the ConatinerRoute helper